core/storage/statistics/gae_models.py

```diff
@@ -824,6 +824,25 @@ def get_multi_versions(cls, exp_id, version_numbers):
         exploration_stats_models = cls.get_multi(entity_ids)
         return exploration_stats_models
 
+    @classmethod
+    def get_multi_stats_models(cls, exp_ids, version_numbers):
```

Comment:
I think the modelling here isn't right; there is a hidden constraint that the two lists are the same length but the two args look independent. A single list of ExpVersionReference domain objects would be better (these are simple domain objects storing just an exp_id and a version_number).

This function is also not tested.

Reply:
Done.